selenium/che-selenium-test/src/main/java/org/eclipse/che/selenium/pageobject/NavigateToFile.java

```diff
@@ -153,12 +154,15 @@ public void selectFileByFullName(String pathName) {
    * @param nameOfFile name of a file for searching
    */
   public void selectFileByName(String nameOfFile) {
-    new WebDriverWait(seleniumWebDriver, REDRAW_UI_ELEMENTS_TIMEOUT_SEC)
-        .until(
-            visibilityOfElementLocated(
-                By.xpath(format(Locators.FILE_NAME_LIST_SELECT, nameOfFile))))
-        .click();
-    actionsFactory.createAction(seleniumWebDriver).doubleClick().perform();
+    WebElement webElement =
+        new WebDriverWait(seleniumWebDriver, REDRAW_UI_ELEMENTS_TIMEOUT_SEC)
+            .until(
+                visibilityOfElementLocated(
+                    By.xpath(format(Locators.FILE_NAME_LIST_SELECT, nameOfFile))));
+
+    webElement.click();
+    WaitUtils.sleepQuietly(500, MILLISECONDS);
```

Comment:
Please, add comment to explain why we should wait so long here.

Reply:
It is timeout between selecting element and double click on it. 0.5 sec is not so long.

Reply:
Yes, you are right.